src/pick.c

```diff
@@ -356,6 +357,7 @@ selected_choice(void)
 			}
 
 			break;
+		case DEL:
```

Comment:
Only one case label is needed here. I suggest you remove the `CTRL_D` value completely and let the `CTRL('D')` binding return `DEL` instead.